sklearn/decomposition/_nmf.py

```diff
@@ -1585,6 +1706,8 @@ def fit_transform(self, X, y=None, W=None, H=None):
         W : ndarray of shape (n_samples, n_components)
             Transformed data.
         """
+        self._validate_params()
```

Comment:
I think that a small comment could be nice here to mention that `fit` calls `fit_transform`. We could also add this information in `fit`.